### DISCUSSION TITLE
Verbindungsprobleme

### DISCUSSION THREAD

**Discussion by Aspire2004** [tbnobody/OpenDTU#971] [Q&A]
Hallo,
habe Verbindungsprobleme, aktuelle Software ist aufm ESP32 installiert.

Verbindung zum Wechselrichter funktioniert nur verliert  er diese nach jeder 2 oder 3 aktualisierung.

weiteres Problem hin und wieder startet die dtu alle paar sekunden neu!

Trenne ich die Stormverbindung läuft diese einige aktualiserungen wieder normal!

Jemand eine Idee?

**Comment by Aspire2004** [comment]
Problem besteht weiterhin, habe schon getetstet ob es mit der vorherigen Version besser läuft aber leider ohne Erfolg!

Wäre super wenn Jemand eine Idee hätte!

**Comment by DPO99** [comment]
Moin, nutzt du nur OpenDTU oder noch weitere DTU's parallel an den gleichen WR? Ansonsten stellt sich noch die Frage nach dem NRF/CMT Modul. Darüber hinaus auch gerne mal das log für ein paar Sekunden laufen lassen und  hier posten, wenn es da zu Fehlern kommt.

**Comment by bitboy00** [comment]
Es kann auch ein Problem mit dem Netzteil sein.

**Comment by Aspire2004** [comment]
Nabend, 
habe die DTU parallel mit der DTU-Wlite verwendet inkl externen NRF Modul!

Habe nun Testweise Ahoy DTU installiert, nun weder Ausetzer oder sonstige Störungen!
An meiner Hardware liegt das Problem also nicht!

Habe heute ebenfalls von so ein Problem in der FB Gruppe gelesen!

OpenDTU scheint wohl im Verbindung mit der original DTU zickig zu sein!

**Comment by tbnobody** [comment]
Das NRF Modul sowie die Firmware auf dem Inverter ist nicht für mehrere DTUs ausgelegt. 
Siehe https://github.com/tbnobody/OpenDTU/wiki/FAQ#q-opendtu-shows-wrongunrealistic-or-just-missing-values

**Comment by Schnurbi** [comment]
Moin, das Thema ist zwar schon alt,
aber ich habe jetzt fast das gleiche Problem.

Nur ist das Problem bei mir auch mit AhoyDTU nicht weg. Dort sieht man zwar schön, dass die Signalstärke für die Verbindung zum WR stark schwankt, aber man sieht nur auf dem Display ab und zu, dass alle paar Sekunden offline ist.
Bei openDTU sieht man sowohl immer wieder im Display dass der WR offline ist (keine Verbindung), als auch in der Weboberfläche.

Ich habe es bereits mit einem anderen Netzteil probiert und auch mit unterschiedlichen Standorten (mal 1,5m vom WR entfernt und mal 10m im nächsten Raum. Auch mal in einem anderen Stockwerk. Es liegt nicht an der Entfernung.

![HM-800_Erreichbarkeit](https://github.com/tbnobody/OpenDTU/assets/129082868/df701983-7a2a-4771-94c2-ad856608d774)


Aus dem Log werde ich leider nicht so richtig schlau, aber vielleicht sieht einer von euch ja was da los ist:

Interrupt received
23:01:11.772 > RX Channel: 23 --> 95 84 61 91 20 84 61 91 20 09 7E C2 FF FF FF FB B0 02 01 D2 7E C7 7E C7 00 00 45 | -80 dBm
23:01:12.105 > RX Period End
23:01:12.105 > Last missing
23:01:12.105 > Request retransmit: 10
23:01:12.105 > TX RequestFrame Channel: 40 --> 15 84 61 91 20 80 18 62 56 8A 67 
23:01:12.413 > RX Period End
23:01:12.413 > Last missing
23:01:12.413 > Request retransmit: 10
23:01:12.413 > TX RequestFrame Channel: 61 --> 15 84 61 91 20 80 18 62 56 8A 67 
23:01:12.499 > RX Period End
23:01:12.499 > Last missing
23:01:12.499 > Request retransmit: 10
23:01:12.499 > TX RequestFrame Channel: 75 --> 15 84 61 91 20 80 18 62 56 8A 67 
23:01:12.724 > RX Period End
23:01:12.724 > Last missing
23:01:12.724 > Request retransmit: 10
23:01:12.724 > TX RequestFrame Channel: 3 --> 15 84 61 91 20 80 18 62 56 8A 67 
23:01:12.927 > RX Period End
23:01:12.927 > Last missing
23:01:12.927 > Request retransmit: 10
23:01:12.927 > TX RequestFrame Channel: 23 --> 15 84 61 91 20 80 18 62 56 8A 67 
23:01:13.335 > RX Period End
23:01:13.335 > Last missing
23:01:13.335 > Retransmit timeout
23:01:13.540 > TX SystemConfigPara Channel: 40 --> 15 84 61 91 20 80 18 62 56 80 05 00 66 45 22 96 00 00 00 00 00 00 00 00 87 45 3D 
23:01:13.629 > RX Period End
23:01:13.629 > All missing
23:01:13.629 > Nothing received, resend whole request
23:01:13.629 > TX SystemConfigPara Channel: 61 --> 15 84 61 91 20 80 18 62 56 80 05 00 66 45 22 96 00 00 00 00 00 00 00 00 87 45 3D 
23:01:13.734 > Interrupt received
23:01:13.809 > RX Channel: 23 --> 95 84 61 91 20 84 61 91 20 81 00 01 03 E8 00 00 03 E8 FF FF FF FF 01 68 7D F8 F9 | -80 dBm
23:01:13.911 > RX Period End
23:01:13.911 > Success
23:01:13.978 > TX GridOnProFilePara Channel: 75 --> 15 84 61 91 20 80 18 62 56 80 02 00 66 45 22 96 00 00 00 00 00 00 00 00 40 43 FB 
23:01:14.037 > Interrupt received
23:01:14.106 > RX Channel: 3 --> 95 84 61 91 20 84 61 91 20 06 00 10 00 00 60 00 00 01 09 E2 0A 5A 02 15 80 01 CF | -80 dBm
23:01:14.199 > RX Period End
23:01:14.199 > Last missing
23:01:14.199 > Request retransmit: 7
23:01:14.199 > TX RequestFrame Channel: 3 --> 15 84 61 91 20 80 18 62 56 87 6A 
23:01:14.289 > RX Period End
23:01:14.289 > Last missing
23:01:14.289 > Request retransmit: 7
23:01:14.289 > TX RequestFrame Channel: 23 --> 15 84 61 91 20 80 18 62 56 87 6A 
23:01:14.445 > RX Period End
23:01:14.445 > Last missing
23:01:14.445 > Request retransmit: 7
23:01:14.445 > TX RequestFrame Channel: 40 --> 15 84 61 91 20 80 18 62 56 87 6A 
23:01:14.529 > RX Period End
23:01:14.529 > Last missing
23:01:14.529 > Request retransmit: 7
23:01:14.529 > TX RequestFrame Channel: 61 --> 15 84 61 91 20 80 18 62 56 87 6A 
23:01:14.588 > RX Period End
23:01:14.588 > Last missing
23:01:14.588 > Request retransmit: 7
23:01:14.588 > TX RequestFrame Channel: 75 --> 15 84 61 91 20 80 18 62 56 87 6A 
23:01:14.668 > RX Period End
23:01:14.668 > Last missing
23:01:14.668 > Retransmit timeout
23:01:14.668 > Fetch inverter: 114184619120
23:01:14.792 > TX RealTimeRunData Channel: 3 --> 15 84 61 91 20 80 18 62 56 80 0B 00 66 45 22 9A 00 00 00 00 00 00 00 00 49 1F AB 
23:01:14.864 > RX Period End
23:01:14.864 > All missing
23:01:14.864 > Nothing received, resend whole request
23:01:14.864 > TX RealTimeRunData Channel: 23 --> 15 84 61 91 20 80 18 62 56 80 0B 00 66 45 22 9A 00 00 00 00 00 00 00 00 49 1F AB 
23:01:14.951 > Interrupt received
23:01:15.012 > RX Channel: 61 --> 95 84 61 91 20 84 61 91 20 01 00 01 00 DB 00 DF 01 E9 00 DB 00 DB 01 E1 00 00 99 | -80 dBm
23:01:15.382 > RX Period End
23:01:15.382 > Last missing
23:01:15.382 > Request retransmit: 2
23:01:15.382 > TX RequestFrame Channel: 40 --> 15 84 61 91 20 80 18 62 56 82 6F 
23:01:15.461 > RX Period End
23:01:15.461 > Last missing
23:01:15.461 > Request retransmit: 2
23:01:15.461 > TX RequestFrame Channel: 61 --> 15 84 61 91 20 80 18 62 56 82 6F 
23:01:15.533 > RX Period End
23:01:15.533 > Last missing
23:01:15.533 > Request retransmit: 2
23:01:15.533 > TX RequestFrame Channel: 75 --> 15 84 61 91 20 80 18 62 56 82 6F 
23:01:15.673 > RX Period End
23:01:15.673 > Last missing
23:01:15.673 > Request retransmit: 2
23:01:15.673 > TX RequestFrame Channel: 3 --> 15 84 61 91 20 80 18 62 56 82 6F 
23:01:15.769 > RX Period End
23:01:15.769 > Last missing
23:01:15.769 > Request retransmit: 2
23:01:15.769 > TX RequestFrame Channel: 23 --> 15 84 61 91 20 80 18 62 56 82 6F 
23:01:15.869 > Interrupt received
23:01:15.969 > RX Channel: 61 --> 95 84 61 91 20 84 61 91 20 02 27 85 00 00 26 E3 00 CC 00 C9 09 5C 13 83 03 9C AF | -80 dBm
23:01:16.029 > RX Period End
23:01:16.029 > Last missing
23:01:16.029 > Retransmit timeout
23:01:16.107 > TX AlarmData Channel: 40 --> 15 84 61 91 20 80 18 62 56 80 11 00 66 45 22 9A 00 00 00 00 00 00 00 00 93 04 70 
23:01:16.798 > RX Period End
23:01:16.798 > All missing
23:01:16.798 > Nothing received, resend whole request
23:01:16.798 > TX AlarmData Channel: 61 --> 15 84 61 91 20 80 18 62 56 80 11 00 66 45 22 9A 00 00 00 00 00 00 00 00 93 04 70 
23:01:17.123 > Interrupt received
23:01:17.205 > RX Channel: 40 --> 95 84 61 91 20 84 61 91 20 09 7E C2 FF FF FF FB B0 02 01 D2 7E C7 7E C7 00 00 45 | -80 dBm
23:01:17.534 > Interrupt received
23:01:17.738 > RX Channel: 3 --> 95 84 61 91 20 84 61 91 20 8C 7E D5 FF FF FF FB 22 53 C7 | -80 dBm
23:01:17.924 > RX Period End
23:01:17.924 > Middle missing
23:01:17.924 > Request retransmit: 1
23:01:17.924 > TX RequestFrame Channel: 75 --> 15 84 61 91 20 80 18 62 56 81 6C 
23:01:17.995 > Interrupt received
23:01:18.089 > RX Channel: 23 --> 95 84 61 91 20 84 61 91 20 01 00 01 B0 01 00 01 3E CB 3E CB 00 00 00 00 B0 02 97 | -80 dBm
23:01:18.352 > RX Period End
23:01:18.352 > Middle missing
23:01:18.352 > Request retransmit: 2
23:01:18.352 > TX RequestFrame Channel: 3 --> 15 84 61 91 20 80 18 62 56 82 6F 
23:01:18.441 > RX Period End
23:01:18.441 > Middle missing
23:01:18.441 > Request retransmit: 2
23:01:18.441 > TX RequestFrame Channel: 23 --> 15 84 61 91 20 80 18 62 56 82 6F 
23:01:18.506 > RX Period End
23:01:18.506 > Middle missing
23:01:18.506 > Request retransmit: 2
23:01:18.506 > TX RequestFrame Channel: 40 --> 15 84 61 91 20 80 18 62 56 82 6F 
23:01:18.607 > RX Period End
23:01:18.607 > Middle missing
23:01:18.607 > Request retransmit: 2
23:01:18.607 > TX RequestFrame Channel: 61 --> 15 84 61 91 20 80 18 62 56 82 6F 
23:01:18.684 > RX Period End
23:01:18.684 > Middle missing
23:01:18.684 > Retransmit timeout
23:01:18.798 > TX GridOnProFilePara Channel: 75 --> 15 84 61 91 20 80 18 62 56 80 02 00 66 45 22 9A 00 00 00 00 00 00 00 00 40 16 A2 
23:01:18.880 > Interrupt received
23:01:18.942 > RX Channel: 40 --> 95 84 61 91 20 84 61 91 20 01 0A 00 20 01 00 0C 08 FC 07 A3 00 0F 09 E2 00 1E 19 | -80 dBm
23:01:19.020 > RX Period End
23:01:19.020 > Last missing
23:01:19.020 > Request retransmit: 2
23:01:19.020 > TX RequestFrame Channel: 3 --> 15 84 61 91 20 80 18 62 56 82 6F 
23:01:19.083 > RX Period End
23:01:19.083 > Last missing
23:01:19.083 > Request retransmit: 2
23:01:19.083 > TX RequestFrame Channel: 23 --> 15 84 61 91 20 80 18 62 56 82 6F 
23:01:19.152 > RX Period End
23:01:19.152 > Last missing
23:01:19.152 > Request retransmit: 2
23:01:19.152 > TX RequestFrame Channel: 40 --> 15 84 61 91 20 80 18 62 56 82 6F 
23:01:19.225 > RX Period End
23:01:19.225 > Last missing
23:01:19.225 > Request retransmit: 2
23:01:19.225 > TX RequestFrame Channel: 61 --> 15 84 61 91 20 80 18 62 56 82 6F 
23:01:19.291 > Interrupt received
23:01:19.585 > RX Channel: 3 --> 95 84 61 91 20 84 61 91 20 02 06 4A 00 14 0A 55 00 14 0A C8 00 0A 09 E2 10 03 B4 | -80 dBm
23:01:19.669 > RX Period End
23:01:19.670 > Last missing
23:01:19.670 > Request retransmit: 3
23:01:19.670 > TX RequestFrame Channel: 75 --> 15 84 61 91 20 80 18 62 56 83 6E 
23:01:19.738 > RX Period End
23:01:19.738 > Last missing
23:01:19.738 > Retransmit timeout
23:01:19.738 > Fetch inverter: 114184619120
23:01:19.888 > TX RealTimeRunData Channel: 3 --> 15 84 61 91 20 80 18 62 56 80 0B 00 66 45 22 9F 00 00 00 00 00 00 00 00 19 20 C1 
23:01:19.950 > Interrupt received
23:01:20.016 > RX Channel: 40 --> 95 84 61 91 20 84 61 91 20 01 00 01 00 D4 00 DB 01 D0 00 D4 00 D8 01 C9 00 00 8F | -80 dBm
23:01:20.103 > RX Period End
23:01:20.103 > Last missing
23:01:20.103 > Request retransmit: 2
23:01:20.103 > TX RequestFrame Channel: 23 --> 15 84 61 91 20 80 18 62 56 82 6F 
23:01:20.172 > Interrupt received
23:01:20.294 > RX Channel: 61 --> 95 84 61 91 20 84 61 91 20 02 27 85 00 00 26 E3 00 CC 00 C9 09 5C 13 83 03 6D 5E | -80 dBm
23:01:20.371 > RX Period End
23:01:20.371 > Last missing
23:01:20.371 > Request retransmit: 3
23:01:20.371 > TX RequestFrame Channel: 40 --> 15 84 61 91 20 80 18 62 56 83 6E 
23:01:20.495 > RX Period End
23:01:20.495 > Last missing
23:01:20.495 > Request retransmit: 3
23:01:20.495 > TX RequestFrame Channel: 61 --> 15 84 61 91 20 80 18 62 56 83 6E 
23:01:20.601 > RX Period End
23:01:20.601 > Last missing
23:01:20.601 > Request retransmit: 3
23:01:20.601 > TX RequestFrame Channel: 75 --> 15 84 61 91 20 80 18 62 56 83 6E 
23:01:20.893 > RX Period End
23:01:20.893 > Last missing
23:01:20.893 > Request retransmit: 3
23:01:20.893 > TX RequestFrame Channel: 3 --> 15 84 61 91 20 80 18 62 56 83 6E 
23:01:21.025 > RX Period End
23:01:21.025 > Last missing
23:01:21.025 > Retransmit timeout
23:01:21.108 > TX AlarmData Channel: 23 --> 15 84 61 91 20 80 18 62 56 80 11 00 66 45 22 9F 00 00 00 00 00 00 00 00 C3 3B 1A 
23:01:21.172 > Interrupt received
23:01:21.229 > RX Channel: 75 --> 95 84 61 91 20 84 61 91 20 01 00 01 B0 01 00 01 3E CB 3E CB 00 00 00 00 B0 02 97 | -80 dBm
23:01:21.308 > Interrupt received
23:01:21.378 > RX Channel: 3 --> 95 84 61 91 20 84 61 91 20 0B 01 D4 7E D0 7E D0 00 00 00 05 B0 02 01 D5 7E D5 83 | -80 dBm
23:01:21.509 > RX Period End
23:01:21.509 > Last missing
23:01:21.509 > Request retransmit: 12
23:01:21.509 > TX RequestFrame Channel: 40 --> 15 84 61 91 20 80 18 62 56 8C 61 
23:01:21.576 > RX Period End
23:01:21.576 > Last missing
23:01:21.576 > Request retransmit: 12
23:01:21.576 > TX RequestFrame Channel: 61 --> 15 84 61 91 20 80 18 62 56 8C 61 
23:01:21.680 > RX Period End
23:01:21.680 > Last missing
23:01:21.680 > Request retransmit: 12
23:01:21.680 > TX RequestFrame Channel: 75 --> 15 84 61 91 20 80 18 62 56 8C 61 
23:01:21.830 > RX Period End
23:01:21.830 > Last missing
23:01:21.830 > Request retransmit: 12
23:01:21.830 > TX RequestFrame Channel: 3 --> 15 84 61 91 20 80 18 62 56 8C 61 
23:01:21.915 > Interrupt received
23:01:21.980 > RX Channel: 23 --> 95 84 61 91 20 84 61 91 20 8C 7E D5 FF FF FF FB 22 53 C7 | -80 dBm
23:01:22.044 > RX Period End
23:01:22.044 > Middle missing
23:01:22.044 > Request retransmit: 2
23:01:22.044 > TX RequestFrame Channel: 23 --> 15 84 61 91 20 80 18 62 56 82 6F 
23:01:22.105 > RX Period End
23:01:22.105 > Middle missing
23:01:22.105 > Retransmit timeout
23:01:22.169 > TX GridOnProFilePara Channel: 40 --> 15 84 61 91 20 80 18 62 56 80 02 00 66 45 22 9F 00 00 00 00 00 00 00 00 10 29 C8 
23:01:22.389 > RX Period End
23:01:22.389 > All missing
23:01:22.389 > Nothing received, resend whole request
23:01:22.389 > TX GridOnProFilePara Channel: 61 --> 15 84 61 91 20 80 18 62 56 80 02 00 66 45 22 9F 00 00 00 00 00 00 00 00 10 29 C8 
23:01:22.550 > Interrupt received
23:01:22.755 > RX Channel: 3 --> 95 84 61 91 20 84 61 91 20 03 13 88 12 C0 00 14 13 EC 00 14 12 8E 00 05 14 50 FD | -80 dBm
23:01:22.951 > RX Period End
23:01:22.951 > Last missing
23:01:22.951 > Request retransmit: 4
23:01:22.951 > TX RequestFrame Channel: 75 --> 15 84 61 91 20 80 18 62 56 84 69 
23:01:23.020 > RX Period End
23:01:23.020 > Last missing
23:01:23.020 > Request retransmit: 4
23:01:23.020 > TX RequestFrame Channel: 3 --> 15 84 61 91 20 80 18 62 56 84 69 
23:01:23.113 > RX Period End
23:01:23.113 > Last missing
23:01:23.113 > Request retransmit: 4
23:01:23.113 > TX RequestFrame Channel: 23 --> 15 84 61 91 20 80 18 62 56 84 69 
23:01:23.241 > RX Period End
23:01:23.241 > Last missing
23:01:23.241 > Request retransmit: 4
23:01:23.241 > TX RequestFrame Channel: 40 --> 15 84 61 91 20 80 18 62 56 84 69 
23:01:23.349 > RX Period End
23:01:23.349 > Last missing
23:01:23.349 > Request retransmit: 4
23:01:23.349 > TX RequestFrame Channel: 61 --> 15 84 61 91 20 80 18 62 56 84 69 
23:01:23.412 > Interrupt received
23:01:23.677 > RX Channel: 3 --> 95 84 61 91 20 84 61 91 20 04 00 05 20 00 00 01 30 03 02 58 09 E2 07 A3 13 9C 1C | -80 dBm
23:01:23.761 > RX Period End
23:01:23.761 > Last missing
23:01:23.761 > Retransmit timeout
23:01:23.761 > Fetch inverter: 114184619120
23:01:23.833 > TX RealTimeRunData Channel: 75 --> 15 84 61 91 20 80 18 62 56 80 0B 00 66 45 22 A3 00 00 00 00 00 00 00 00 18 21 FD

**Reply by ckvsoft** [reply]
Hallo
Das gleiche Problem habe ich seit Gestern auch. HM-1500

RX Period End
12:56:38.034 > All missing
12:56:38.034 > Nothing received, resend count exeeded
12:56:38.200 > TX SystemConfigPara Channel: 75 --> 15 91 10 29 04 80 11 42 48 80 05 00 66 61 95 DF 00 00 00 00 00 00 00 00 29 16 D5 
12:56:38.407 > RX Period End
12:56:38.407 > All missing
12:56:38.407 > Nothing received, resend whole request
12:56:38.407 > TX SystemConfigPara Channel: 3 --> 15 91 10 29 04 80 11 42 48 80 05 00 66 61 95 DF 00 00 00 00 00 00 00 00 29 16 D5 
12:56:38.715 > RX Period End
12:56:38.715 > All missing
12:56:38.715 > Nothing received, resend whole request
12:56:38.715 > TX SystemConfigPara Channel: 23 --> 15 91 10 29 04 80 11 42 48 80 05 00 66 61 95 DF 00 00 00 00 00 00 00 00 29 16 D5 
12:56:38.837 > Websocket: [/livedata][5] disconnect
12:56:38.898 > RX Period End
12:56:38.898 > All missing
12:56:38.898 > Nothing received, resend whole request
12:56:38.898 > TX SystemConfigPara Channel: 40 --> 15 91 10 29 04 80 11 42 48 80 05 00 66 61 95 DF 00 00 00 00 00 00 00 00 29 16 D5 
12:56:39.229 > RX Period End
12:56:39.229 > All missing
12:56:39.229 > Nothing received, resend whole request
12:56:39.229 > TX SystemConfigPara Channel: 61 --> 15 91 10 29 04 80 11 42 48 80 05 00 66 61 95 DF 00 00 00 00 00 00 00 00 29 16 D5 
12:56:39.332 > RX Period End
12:56:39.332 > All missing
12:56:39.332 > Nothing received, resend count exeeded
12:56:39.332 > Fetch inverter: 116191102904
12:56:39.332 > Request SystemConfigPara
12:56:39.356 > TX RealTimeRunData Channel: 75 --> 15 91 10 29 04 80 11 42 48 80 0B 00 66 61 95 E7 00 00 00 00 00 00 00 00 26 2A D0 
12:56:39.910 > RX Period End
12:56:39.910 > All missing
12:56:39.910 > Nothing received, resend whole request
12:56:39.910 > TX RealTimeRunData Channel: 3 --> 15 91 10 29 04 80 11 42 48 80 0B 00 66 61 95 E7 00 00 00 00 00 00 00 00 26 2A D0 
12:56:40.555 > RX Period End
12:56:40.555 > All missing
12:56:40.555 > Nothing received, resend whole request
12:56:40.555 > TX RealTimeRunData Channel: 23 --> 15 91 10 29 04 80 11 42 48 80 0B 00 66 61 95 E7 00 00 00 00 00 00 00 00 26 2A D0 
12:56:41.274 > RX Period End
12:56:41.274 > All missing
12:56:41.274 > Nothing received, resend whole request
12:56:41.274 > TX RealTimeRunData Channel: 40 --> 15 91 10 29 04 80 11 42 48 80 0B 00 66 61 95 E7 00 00 00 00 00 00 00 00 26 2A D0 
12:56:41.684 > RX Period End
12:56:41.684 > All missing
12:56:41.684 > Nothing received, resend whole request
12:56:41.684 > TX RealTimeRunData Channel: 61 --> 15 91 10 29 04 80 11 42 48 80 0B 00 66 61 95 E7 00 00 00 00 00 00 00 00 26 2A D0 
12:56:42.092 > RX Period End
12:56:42.092 > All missing
12:56:42.092 > Nothing received, resend count exeeded
12:56:42.104 > TX AlarmData Channel: 75 --> 15 91 10 29 04 80 11 42 48 80 11 00 66 61 95 E7 00 00 00 00 00 00 00 00 FC 31 0B 
12:56:42.872 > RX Period End
12:56:42.872 > All missing
12:56:42.872 > Nothing received, resend whole request
12:56:42.872 > TX AlarmData Channel: 3 --> 15 91 10 29 04 80 11 42 48 80 11 00 66 61 95 E7 00 00 00 00 00 00 00 00 FC 31 0B 
12:56:43.837 > RX Period End
12:56:43.837 > All missing
12:56:43.837 > Nothing received, resend whole request
12:56:43.837 > TX AlarmData Channel: 23 --> 15 91 10 29 04 80 11 42 48 80 11 00 66 61 95 E7 00 00 00 00 00 00 00 00 FC 31 0B 
12:56:44.549 > RX Period End
12:56:44.549 > All missing
12:56:44.549 > Nothing received, resend whole request
12:56:44.549 > TX AlarmData Channel: 40 --> 15 91 10 29 04 80 11 42 48 80 11 00 66 61 95 E7 00 00 00 00 00 00 00 00 FC 31 0B 
12:56:45.269 > RX Period End
12:56:45.269 > All missing
12:56:45.269 > Nothing received, resend whole request
12:56:45.269 > TX AlarmData Channel: 61 --> 15 91 10 29 04 80 11 42 48 80 11 00 66 61 95 E7 00 00 00 00 00 00 00 00 FC 31 0B 
12:56:46.009 > RX Period End
12:56:46.009 > All missing
12:56:46.009 > Nothing received, resend count exeeded
12:56:46.092 > TX SystemConfigPara Channel: 75 --> 15 91 10 29 04 80 11 42 48 80 05 00 66 61 95 E7 00 00 00 00 00 00 00 00 E8 25 1F 
12:56:46.392 > RX Period End
12:56:46.392 > All missing
12:56:46.392 > Nothing received, resend whole request
12:56:46.392 > TX SystemConfigPara Channel: 3 --> 15 91 10 29 04 80 11 42 48 80 05 00 66 61 95 E7 00 00 00 00 00 00 00 00 E8 25 1F 
12:56:46.700 > RX Period End
12:56:46.700 > All missing
12:56:46.700 > Nothing received, resend whole request
12:56:46.700 > TX SystemConfigPara Channel: 23 --> 15 91 10 29 04 80 11 42 48 80 05 00 66 61 95 E7 00 00 00 00 00 00 00 00 E8 25 1F 
12:56:46.847 > RX Period End
12:56:46.847 > All missing
12:56:46.847 > Nothing received, resend whole request
12:56:46.847 > TX SystemConfigPara Channel: 40 --> 15 91 10 29 04 80 11 42 48 80 05 00 66 61 95 E7 00 00 00 00 00 00 00 00 E8 25 1F 
12:56:47.111 > RX Period End
12:56:47.111 > All missing
12:56:47.111 > Nothing received, resend whole request
12:56:47.111 > TX SystemConfigPara Channel: 61 --> 15 91 10 29 04 80 11 42 48 80 05 00 66 61 95 E7 00 00 00 00 00 00 00 00 E8 25 1F 
12:56:47.316 > RX Period End
12:56:47.316 > All missing
12:56:47.316 > Nothing received, resend count exeeded
12:56:47.316 > Fetch inverter: 116191102904
12:56:47.316 > Request SystemConfigPara
12:56:47.327 > TX RealTimeRunData Channel: 75 --> 15 91 10 29 04 80 11 42 48 80 0B 00 66 61 95 EF 00 00 00 00 00 00 00 00 E6 4D 7F 
12:56:47.857 > RX Period End
12:56:47.857 > All missing
12:56:47.857 > Nothing received, resend whole request
12:56:47.857 > TX RealTimeRunData Channel: 3 --> 15 91 10 29 04 80 11 42 48 80 0B 00 66 61 95 EF 00 00 00 00 00 00 00 00 E6 4D 7F 
12:56:48.442 > RX Period End
12:56:48.442 > All missing
12:56:48.442 > Nothing received, resend whole request
12:56:48.442 > TX RealTimeRunData Channel: 23 --> 15 91 10 29 04 80 11 42 48 80 0B 00 66 61 95 EF 00 00 00 00 00 00 00 00 E6 4D 7F 
12:56:49.059 > RX Period End
12:56:49.059 > All missing
12:56:49.059 > Nothing received, resend whole request
12:56:49.059 > TX RealTimeRunData Channel: 40 --> 15 91 10 29 04 80 11 42 48 80 0B 00 66 61 95 EF 00 00 00 00 00 00 00 00 E6 4D 7F 
12:56:49.570 > RX Period End
12:56:49.570 > All missing
12:56:49.570 > Nothing received, resend whole request
12:56:49.570 > TX RealTimeRunData Channel: 61 --> 15 91 10 29 04 80 11 42 48 80 0B 00 66 61 95 EF 00 00 00 00 00 00 00 00 E6 4D 7F 
12:56:50.018 > RX Period End
12:56:50.018 > All missing
12:56:50.018 > Nothing received, resend count exeeded
12:56:50.044 > TX AlarmData Channel: 75 --> 15 91 10 29 04 80 11 42 48 80 11 00 66 61 95 EF 00 00 00 00 00 00 00 00 3C 56 A4 
12:56:50.847 > RX Period End
12:56:50.847 > All missing
12:56:50.847 > Nothing received, resend whole request
12:56:50.847 > TX AlarmData Channel: 3 --> 15 91 10 29 04 80 11 42 48 80 11 00 66 61 95 EF 00 00 00 00 00 00 00 00 3C 56 A4 
12:56:51.841 > RX Period End
12:56:51.841 > All missing
12:56:51.841 > Nothing received, resend whole request
12:56:51.841 > TX AlarmData Channel: 23 --> 15 91 10 29 04 80 11 42 48 80 11 00 66 61 95 EF 00 00 00 00 00 00 00 00 3C 56 A4

**Reply by Schnurbi** [reply]
Bei mir lag es an der Hardware. Habe vom Verkäufer eine neue openDTU bekommen, die auf Anhieb problelos lief.

**Reply by ckvsoft** [reply]
> Bei mir lag es an der Hardware. Habe vom Verkäufer eine neue openDTU bekommen, die auf Anhieb problelos lief.

Das kann ich ja mal mit einer neuen versuchen, meine wird sicher nicht mehr umgetauscht, dafür ist die wohl schon zu alt

**Reply by Schnurbi** [reply]
Würde ich tatsächlich empfehlen. Ich hab vom Verkäufer leider kein Rückmeldung welches Teil das Problem war. Er hatte erst nach eine genauen Neschreibung von mir den Fehler nachvollziehen können, da er sie ursprünglich mit ahoyDTU ausgeliefert hatte und da fehlt die Offline-Anzeige auf dem Display und auch auf dem Webinterface waren die Unterbrechungen da nicht so gut zu erkennen wie bei openDTU.
